extensions/guacamole-auth-cas/src/main/resources/translations/ca.json

```diff
@@ -0,0 +1,12 @@
+{
+
+    "DATA_SOURCE_CAS" : {
+        "NAME" : "CAS SSO Backend"
+    },
+
+    "LOGIN" : {
+        "FIELD_HEADER_TICKET"        : "",
```

Comment:
You can leave out empty strings like this, since they will inherit from the English translation.

Reply:
Empty strings removed